client/lib/analytics/signup.js

```diff
@@ -65,6 +65,8 @@ export function recordSignupComplete(
 		);
 	}
 
+	const signUpDomainOrigin = window.localStorage.getItem( 'SIGNUP_DOMAIN_ORIGIN' );
```

Comment:
If this is not set, do we want to keep it `null`, or can we fall back to something else? E.g. via null coalescing `??` 

It's more of a definition (Tracks events support blank values). It works the way it is.